testing/ef_tests/check_all_files_accessed.py

```diff
@@ -47,7 +47,7 @@
     "bls12-381-tests/hash_to_G2",
     "tests/.*/eip6110",
     "tests/.*/whisk",
-    "tests/.*/eip7594",
+    "tests/.*/fulu",
```

Comment:
Why do we exclude fulu?

Reply:
This PR still runs alpha-8 and the tests won't pass until we merge custody group (#6736) and electra PRs (#6488)